Index: app/src/main/java/com/mdwg2/covid19/QuizActivity.java
===================================================================
--- app/src/main/java/com/mdwg2/covid19/QuizActivity.java	(revision e9ea3d1b1e5f3ce8fa1c4b558ca39366db096cf8)
+++ app/src/main/java/com/mdwg2/covid19/QuizActivity.java	(revision e9ea3d1b1e5f3ce8fa1c4b558ca39366db096cf8)
@@ -1,30 +0,0 @@
-package com.mdwg2.covid19;
-
-import androidx.appcompat.app.AppCompatActivity;
-
-import java.awt.Button;
-
-import javax.swing.text.View;
-import android.content.Intent;
-import android.os.Bundle;
-import android.widget.Toast;
-
-
-
-public class QuizActivity extends AppCompatActivity implements View.OnClickListener  {
-    Button b;
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main);
-        b = findViewById(R.id.bouton);
-        b.setOnClickListener(this);
-    }
-    public void onClick(View v) {
-        if (v.getId() == R.id.bouton) {
-            Toast.makeText(getApplicationContext(), "vous avez cliquer sur le bouton Suivant", Toast.LENGTH_SHORT).show();
-            Intent sondage = new Intent(this, activity_sondage_main.Class);
-            startActivity(sondage);
-        }
-    }
-}
